site/_includes/partials/post-banner.njk

```diff
@@ -0,0 +1,3 @@
+<div class="post-banner">
+  This content is outdated and no longer accurate. <a href={{ newMaterialURL }}>New material available</a>.
```

Comment:
Please don't use fully qualified URLs in the frontmatter. They may end up broken if the content is removed/moved. Please use the path to the document instead and derive the URL via 11ty filters.

Reply:
To be more specific: we have a helper `findByPath` that you could export from /site/_data/helpers.js. In the templates that should allow you to construct a URL by using `helpers.findByPath(new_material).url` then, which should be less error-prone.

Reply:
I have fixed this 😃